src/locales/en-AU.json

```diff
@@ -501,6 +501,7 @@
   "mnoe_admin_panel.components.mno-product-selector.create_order": "Create an order",
   "mnoe_admin_panel.components.mno-product-selector.cancel": "Cancel",
   "mnoe_admin_panel.components.mno-product-selector.categories.select_all": "All",
+  "mnoe_admin_panel.components.mno-product-selector.billing_details_req": "Before adding products to this account, this organization needs to update their billing information.",
```

Comment:
The wording could be simplified a bit, something along the lines of:
> This organization needs to update their billing information before adding products.
> Before adding products to this organization, they need to update their billing information

Reply:
Updated wording and squashed into original commit